app/policies/bulk_upload_policy.rb

```diff
@@ -0,0 +1,26 @@
+class BulkUploadPolicy
```

Comment:
It's probably not that important, but would it be worth creating an `ApplicationPolicy` class that this class inherits from?

Reply:
might need one in the future but don't really see what i would add to the base class if i added one right now?

Reply:
Fair enough. I was wondering about whether we could write something more generic in an `ApplicationPolicy` class which the bulk upload stuff could be a special case of. Not sure whether that would make sense here - even if it did, it probably wouldn't be worth doing until we had other policies doing something similar to the bulk upload policy.